cmd/flux/cmd/test.go

```diff
@@ -0,0 +1,151 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/ast/edit"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute/executetest"
+	"github.com/influxdata/flux/execute/table"
+	"github.com/influxdata/flux/fluxinit"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/parser"
+	"github.com/influxdata/flux/runtime"
+	"github.com/spf13/cobra"
+)
+
+type Test struct {
+	ast *ast.Package
+	Err error
+}
+
+func NewTest(ast *ast.Package) Test {
+	return Test{
+		ast: ast,
+	}
+}
+
+func (t *Test) Name() string {
+	return t.ast.Files[0].Name
+}
+
+func (t *Test) Run() {
+	jsonAST, Err := json.Marshal(t.ast)
+	if Err != nil {
+		t.Err = Err
+		return
+	}
+	c := lang.ASTCompiler{AST: jsonAST}
+
+	ctx := executetest.NewTestExecuteDependencies().Inject(context.Background())
+	program, Err := c.Compile(ctx, runtime.Default)
+	if Err != nil {
+		t.Err = errors.Wrap(Err, codes.Invalid, "failed to compile")
+	}
+
+	alloc := &memory.Allocator{}
+	result, Err := program.Start(ctx, alloc)
```

Comment:
`Err` -> `err` (same comment everywhere this is done)

Reply:
Yeah, that was a rename issue. At one point, I had `Test.err` and decided to make it public (missing the pattern you suggested above) and the editor just decided to rename everything called `err` to `Err`.